js/projects_info/projects_info.js

```diff
@@ -9,8 +9,15 @@ var projectsInfo = [
     'name': "Scout-App",
     'imageLocation': './images/project_images/scout-app.svg',
     'link': 'http://scout-app.io/',
-    'description': "Scout-App, the easiest way to use Sass!",
-    'languages': ["jquery", "javascript", "nodejs", "sass", "html5"]
+    'description': 'Scout-App, the easiest way to use Sass!',
+    'languages': ["html5", "javascript", "jquery", "nodejs", "sass"]
```

Comment:
Scout-App is primarily built on top of NW.js, formerly known as Node-Webkit. DevIcon hasn't been updated to the new name yet, but there is an icon for it.

Though NW.js is a runtime environment, not a language. and jQuery is a library and not a language. The key `"languages"` should be changed to `"techStack"` which is more reflective of what you actually listing.